prog/vm/nexus.rb

```diff
@@ -336,6 +336,16 @@ def write_params_json
       amount: vm.cores
     )
 
+    storage_volumes.each do |vol|
+      BillingRecord.create_with_id(
+        project_id: project.id,
+        resource_id: vm.id,
+        resource_name: vm.name,
+        billing_rate_id: BillingRate.from_resource_properties("VmStorage", vm.family, vm.location)["id"],
+        amount: vol["size_gib"]
```

Comment:
Should we change `resource_id` to storage volume ID, or should we rename `resource_name`? At present, if a VM has two disk volumes of the same size, they are indistinguishable.

Reply:
I changed the resource_name and added disk_index to be able to distinguish different disks.